### DISCUSSION TITLE
Modernizing Tree Detection with Advanced Object Detection Models

### DISCUSSION THREAD

**Discussion by ire-mide1** [weecology/DeepForest#640] [General]
Hello, I am a computer engineering student at Federal University of Technology Akure and a passionate machine learning enthusiast. I am excited to contribute to Modernizing Tree Detection with Advanced Object Detection Models project by evaluating and updating the existing 1-stage object detection backbone in the DeepForest model for tree detection. I am eager to work with the mentors and explore transitioning to vision transformers to enhance the project's capabilities

**Comment by bw4sz** [comment]
Welcome @ire-mide1, happy to discuss. @henrykironde #628

**Reply by henrykironde** [reply]
@ire-mide1 Welcome aboard! As Ben mentioned, please take a look at #628 and share your thoughts. Don't hesitate to reach out if you have any questions along the way. We're here to help!

**Comment by ire-mide1** [comment]
Hello @bw4sz @henrykironde , I hope you are good?

I have successfully setup deepforest repo on my local device. I have also solved a good first issue .Please review my Pull Request.
